### PR TITLE
Fixes #1859 - Adds tests and data query for populating the view.

### LINKED ISSUE #1859: Triage dashboard query information
Context: #1857 
Previous issue: #1858 

We need two things: 

1. Total count of needstriage issues https://github.com/webcompat/web-bugs/milestones
2. Total count of needstriage issues older than 48 hours. (`is:issue milestone:needstriage created:<2017-10-30` on `2017-11-01`) **AND** without a `status-needinfo` label.
3. The list of issues in needstriage and/or only the ones which are older than 48 hours 

* [x] Create tests for the number being queried on GitHub
* [x] Create tests for checking the list of issues being returned by GitHub as a JSON
* [x] Create test to make sure needinfo is not part of the more than 48hours issues.
* [x] Create function for querying the two numbers we need
* [x] Create function for getting the list of these older issues (Maybe it's already in the previous request.)

### PR DESCRIPTION
**Pull request by karlcow**
That's it. It's done.

Issue #1859 is complete. The dashboard is operational. 
Now we need to sprinkle a bit of beauty into it and play with JS/CSS for filtering the data in #1861 
But as it is now. It is already mergeable on master.

### CODE REVIEW

**Comment by zoepage** [comment]
Just saw the build is failing? Maybe worth checking before merging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 2. New features: affect minor number
 3. API endpoint changes or dependency version updates: affect major number
 
+## 6.0.2 - 2017-11-01
```
Comment (miketaylr) [inline CHANGELOG.md]: Can you rebase your branch against the latest master?

File: webcompat/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```
Comment (miketaylr) [inline webcompat/dashboard.py]:
```
$ pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example

webcompat/dashboard.py:35:24: E241 multiple spaces after ':'

webcompat/dashboard.py:38:13: E123 closing bracket does not match indentation of opening bracket's line

webcompat/dashboard.py:60:13: W503 line break before binary operator

The command "pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example" failed and exited with 1 during .
```

Can you fix that up @karlcow?

**Comment by miketaylr** [comment]
LGTM to merge once the pep8 stuff is cleaned up and rebased against master.

**Comment by miketaylr** [comment]
(the dashboard branch will need to be rebased, that is)

File: webcompat/dashboard.py (hunk 2 above)
Comment (karlcow) [reply webcompat/dashboard.py]: This has been fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 2. New features: affect minor number
 3. API endpoint changes or dependency version updates: affect major number
 
+=======
```
Comment (karlcow) [inline CHANGELOG.md]: huh what's that :)
hmmm did I messed up my rebase on dashboard. Let me see.

**Comment by karlcow** [comment]
@miketaylr 

* pep8 stuff fixed.
* I had to wrestle with remote dashboard and local 1859/1 but it seems to be ok?

**Comment by karlcow** [comment]
hmm why do I have more commits than before ø_0

**Comment by karlcow** [comment]
ah maybe because when I worked on 1859 I started by pull 1858 for having an up to date repo.

**Comment by miketaylr** [comment]
@karlcow I see some conflicts. Can you take a look?

**Comment by karlcow** [comment]
@miketaylr Merge conflicts resolved.

**Comment by karlcow** [comment]
* ok no more conflicts. 
* changes made
* all checks have passed

**Comment by miketaylr** [comment]
Lovely. Thanks @karlcow!

**Pull request merged**